sfdx-source/apex-common/main/classes/fflib_SecurityUtils.cls

```diff
@@ -52,14 +52,12 @@ public class fflib_SecurityUtils
     	private CrudException(OperationType operation, Schema.SObjectType objectType){
     		this.m_operation = operation;
     		this.m_objectType = objectType;
-    		if(operation == OperationType.CREATE)
-    			this.setMessage(System.Label.fflib_security_error_object_not_insertable);
-    		else if(operation == OperationType.READ)
-    			this.setMessage(System.Label.fflib_security_error_object_not_readable);
-    		else if(operation == OperationType.MODIFY)
-    			this.setMessage(System.Label.fflib_security_error_object_not_updateable);
-    		else if(operation == OperationType.DEL)
-    			this.setMessage(System.Label.fflib_security_error_object_not_deletable);
+		Map<OperationType, String> messagesByOperation= new Map<OperationType, String>{OperationType.CREATE=> System.Label.fflib_security_error_object_not_insertable,
```

Comment:
@clebrsonn this is more of a functional change as opposed to a change for readability.  Was that the intent?  If so, what is the benefit that you see by doing this?